### ISSUE TITLE
Can't print spreadsheet nor export to PDF

### ISSUE THREAD

**Issue by muchachagrande** [CollaboraOnline/online#8931]
### Describe the Bug
When trying to export a PDF of a spreadsheet, the exported file is zero bytes. I think that this is the reason of the second problem that it can't print the spreadsheet.

### Steps to Reproduce
1. Open or create a spreadsheet
2. Go to file -> print
3. Nothing happens

OR

1. Open or create a spreadsheet
2. go to file -> download as PDF
3. A zero length PDF is downloaded and opened on the browser

### Expected Behavior
When exporting a PDF the expected behaviour is that the file should be downloaded and then opened in the browser.

When printing, the print dialogue should open and allow save a PDF or print. 

### Actual Behavior
When exporting a PDF a zero size file is being downloaded and opened.

When printing, nothing happens.

### Screenshots
No screenshot

### Desktop
(Please complete the following information)
 - **Collabora version:** 24.04.1.3
 - **OS and version:** Ubuntu 22.04
 - **Browser and version:** Firefox 125.0.3

### Additional Context
Collabora is on docker in the same VM as the Nextcloud 28.0.5

**Comment by muchachagrande** [comment]
Well, an update to the description: using the compact view, the print button works. Still the PDF is exported as an empty file.

**Comment by timur-g** [comment]
Print is a common issue as seen in #8969 so let us track it there. 
For download, there are many closed and open tickets, also those for docker, so please search. Probably relevant if that is rooless or rootfull docker.

**Comment by muchachagrande** [comment]
Hi, @timur-g,
as you pointed out #8969 is the exact problem I have, so you are already tracking it.
The second problem I'm referring was reported on some tickets. One of them I found is https://github.com/CollaboraOnline/online/issues/5088#issuecomment-2058008461
The solution referenced in this post solved the issue for me. I had to add --o:security.capabilities=false on the extra_params of the docker container and PDFs started to export correctly.
Now I'll try to understand why and what other implications have using that option.
